### ISSUE TITLE
[TASK] Unable to test My Courses without Course data or test accounts with Course data

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5796]
### Description
_I need to be able to test My Canvas Courses section as a Student that uses assistive technology._
_Example: As a screen reader user (of all types, specifically student), I want to be able to have Course data so that I can complete my regression testing of 7.3.51 version prior to the April ADA deadline._

### Acceptance Criteria
_User will have Canvas course data. 
_User will have past/present/future courses ideally

I'm not sure who populates this. All test accounts in bitwarden were locked or did not have the proper data. Thus, submitting this request.

**Comment by acovher2** [comment]
I don't want to mark the priority of SS but this is simply blocking this section's testing in the app. Do I need a separate ticket to get My Courses populated too? 

Thanks.

**Comment by vburgett** [comment]
We haven't been able to accommodate this need. In the past, we have worked directly with students to test. @ToddNelson do you have anything else to add?

**Comment by ToddNelson** [comment]
This is a challenge we have not yet been able to resolve. We are unable to generate or use fake course data for testing purposes. While Misho has student NetID flagged credentials, he does not have active course enrollments available for testing scenarios.

At this point, the most viable option may be to identify a student from Design's feedback group who would be willing to partner with Amanda to execute the necessary tests using their live course data.

**Comment by ToddNelson** [comment]
Amanda... is this a viable solution?

**Comment by acovher2** [comment]
@ToddNelson  This is a viable solution but I am just seeing it. Am I allowed to follow through with testing that?
